uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/leave-request-instance_test.js

```diff
@@ -0,0 +1,75 @@
+define([
+  'mocks/apis/leave-request-api-mock',
+  'leave-absences/shared/models/leave-status-id-model',
```

Comment:
`leave-absences/shared/models/instances/leave-request-instance` is already requiring this, why are you requiring it again?